package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "newrelic",
-  "version": "9.6.0",
+  "version": "9.6.1",
```

Comment:
the scope of this release is a new feature not bug fix. I would change to 9.7.0 and also in package-lock.json

Reply:
Alright, let's just do a new PR.

Reply:
you don't have to do a new PR, just edit these 2 files